docs/deploying.md

```diff
@@ -1,40 +1,48 @@
 # Deploying
 
-When time comes to share your project, you have many options for deploying it for others to see. Framework is compatible with many static site hosts and automation environments. In this guide we’ll focus on deploying to Observable manually, then with GitHub Actions.
+You can host your built Framework project on any static site hosting service, or self-host it with any static site server. This guide covers deploying to [Observable](https://observablehq.com), which is the easiest way to host your Framework project as support is built-in. We’ll also cover setting up automated deploys with GitHub Actions.
 
-## Manually deploying to Observable
+<div class="tip">
 
-If you don’t already have a project to deploy, you can create one by following [getting-started](./getting-started). First, make sure that your project builds without error:
+If you don’t already have a project ready to deploy, create one by following our [Getting started guide](./getting-started).
+
+</div>
+
+## Manual deploys
+
+First, make sure that your project builds without error:
 
 ```sh
-$ npm run build
+npm run build
 ```
 
-Once that is done you can deploy to Observable with the command
+Once that is done you can deploy to Observable:
 
 ```sh
-$ npm run deploy
+npm run deploy
 ```
 
-The first time you run this command, you will be prompted for details needed to set up the project on the server, such as the project's _slug_ (which determines its URL), and the access level. If you don’t already have an Observable account or aren’t signed in, this command will also guide you through setting that up.
+The first time you deploy a project, you will be prompted to configure the project’s _slug_ (which determines its URL), access level, and other details. If you don’t yet have an Observable account or aren’t signed-in, you will also be prompted to sign-up or sign-in.
 
-When the deploy command finishes, it prints a link to observablehq.cloud where you can view your deployed project. If you choose “private” as the access level, you can now share that link with anyone who is a member of  your workspace. If you chose “public”, you can share that link with anyone and they’ll be able to see your Framework project.
+When the deploy command finishes, it prints a link to observablehq.cloud where you can view your deployed project. If you choose *private* as the access level, that link will only be accessible to members of your Observable workspace. (You can invite people to your workspace by going to observablehq.com.) If you chose *public*, you can share your project link with anyone.
```

Comment:
It’s important to explain that to invite people to view a _project_ (user goal), you invite them to join the associated _workspace_ (user action). Eventually, we want it to feel more like inviting people directly to the project, but for now you have to know that the workspace is what controls access.